src/igvfd/tests/fixtures/schemas/primary_cell.py

```diff
@@ -8,19 +8,21 @@ def primary_cell(testapp, other_lab, award, human_donor, sample_term_pluripotent
         'lab': other_lab['@id'],
         'source': other_lab['@id'],
         'donors': [human_donor['@id']],
-        'biosample_term': sample_term_pluripotent_stem_cell['@id']
+        'biosample_term': sample_term_pluripotent_stem_cell['@id'],
+        'virtual': True
     }
     return testapp.post_json('/primary_cell', item, status=201).json['@graph'][0]
 
 
 @pytest.fixture
-def pooled_from_primary_cell(testapp, lab, award, source, human_donor, sample_term_pluripotent_stem_cell):
+def pooled_from_primary_cell(testapp, lab, award, source, human_donor, sample_term_pluripotent_stem_cell, primary_cell):
```

Comment:
why do we add "primary_cell" to the parameters?

Reply:
Removed.